Resources/gulpfile.js

```diff
@@ -0,0 +1,21 @@
+const gulp = require('gulp');
+const babel = require('gulp-babel');
+const rename = require('gulp-rename');
+const uglify = require('gulp-uglify');
+const wrap = require('gulp-wrap');
+
+gulp.task('js', () => {
+    gulp.src('js/router.js')
+        .pipe(babel())
+        .pipe(wrap({ src: 'js/router.template.js' }))
+        .pipe(gulp.dest('public/js'))
+        .pipe(rename((path) => {
+            path.extname = '.min.js';
+        }))
+        .pipe(uglify())
+        .pipe(gulp.dest('public/js'));
+});
+
+gulp.task('default', () => {
+    gulp.start(['js']);
```

Comment:
missing return here too.

And btw, I don't think we need 2 different tasks.

Reply:
Return added in a soon-to-be pushed commit.

Having two tasks isn't strictly necessary and I'm reasonably new to using Gulp so am happy to remove the second if you would prefer? My reading suggests that the 'standard' is to have a 'default' task which essentially acts as a 'controller' calling the other tasks which actually do the doing (so in a future version for example a 'css' task could be added)